pages/src/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "env": {
+    "browser": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "script"
+  }
+}
```

Comment:
won't this file override the parent eslintrc and remove the custom rules you set up? If the only difference is adding `env.browser` then I think it's safe to merge in